MySQL.sql

```diff
@@ -0,0 +1,16 @@
+CREATE SCHEMA zillowDescription;
+
+USE zillowDescription;
+
+CREATE TABLE houses (
+	id INT NOT NULL PRIMARY KEY AUTO_INCREMENT,
+  street varchar(255),
+  city varchar(255),
+  state varchar(5),
+  zipcode varchar(10),
+  description text,
+  price INT
+);
+
+CREATE INDEX idx_id
+ON houses (id);
```

Comment:
Seems that primary keys are already indexed.  Mainly just a small comment for future reference rather than a change.
https://stackoverflow.com/questions/1071180/is-the-primary-key-automatically-indexed-in-mysql

Reply:
Gotcha. Thanks for the comment there. Will implement that in future projects.